File: backend/agropec/model/Bovine.py
from peewee import *

from .Category import Category
from .Base import BaseModel
from .Earring import Earring

class Bovine(BaseModel):
    id = AutoField()
    sex = SmallIntegerField()
    weight = FloatField()
    category = ForeignKeyField(Category, backref='id')
    earring = ForeignKeyField(Earring, backref='id')

File: frontend/src/model/Bovine.ts
import type { Category } from "./Category";
import type { Earring } from "./Earring";

export class Bovine {
    id: number | null | undefined;
    sex: number | null | undefined;
    weight: number | null | undefined;
    category: Category | null | undefined;
    earring: Earring | null | undefined;
}
